index.html

```diff
@@ -90,6 +92,227 @@ <h2>Let's connect</h2>
+                                </svg>
+
+                            </a>
+                        </li>
+                    </ul>
+
+                    <div class="mt-1">
+                        <a class="btn" href="#">Get my resume</a>
+                    </div>
+
+                </section>
+
+                <ul class="drop-buttons">
+                    <li class="drop-button">
+                        <p class="title">Languages</p>
+                        <ul class="languages extra-menu">
+                            <li>
+                                <img src="../images/javascript.png" alt="javascript icon" />
+                                <p>JavaScript</p>
+                            </li>
+                            <li>
+                                <img src="../images/html.png" alt="javascript icon" />
+                                <p>HTML</p>
```

Comment:
- [ ] Please for best practice's sake, Why don't you consider using `span` instead of using the `p` element since this is not a paragraph? :smile: